### PR TITLE
Api600 volumes

### PR DESCRIPTION
**Pull request by madhav-bharadwaj**
### Description
Api600 Support for Volumes

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass (`$ rake test`).
- [X] New functionality has been documented in the README if applicable.
  - [X] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [X] New endpoints supported are updated in the endpoints-support.md file.
- [X] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/api600/volume.rb [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
```
Comment (soodpr) [inline examples/api600/volume.rb]: Not sure why we have added a new file here. Can't we just reuse the existing example

Comment (madhav-bharadwaj) [reply examples/api600/volume.rb]: No we cannot use existing example as the request body for api300 and api500,600 are different. So there is a different example for API500 under examples/api500 folder and therefore I have created a separate file for api600 going by the directory name. We can come to a consensus as to how we can handle this as both api500 and api600 have similar examples and dont need 2 different files.

Comment (soodpr) [reply examples/api600/volume.rb]: yes. Thats what i meant. if there is no change between 500 and 600, then lets keep same example file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ This release adds support to OneView Rest API version 600 for the hardware varia
    - SAS Logical Interconnect Group
    - Server Hardware
    - Server Profile Template
-
+   - Volume
+
```
Comment (jyothis-gireesan) [inline CHANGELOG.md]: You can remove the unwanted spaces in the extra line.

**Pull request merged**